dockerfile-buildx-to-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for i in $(find . -name Dockerfile.buildx -print); do
+    out=$(echo "$i" | sed 's/x$//')
+    perl -ne 'next if /^# Note:.*intended.*buildx/;s/--mount\S+//;print' <"$i" >"${out}"
```

Comment:
May want to comment at the top of the file that this simply/blindly removes mount options and may need to be extended for other incompatibilities between build and buildx.

Reply:
Done.